test/runtests.jl

```diff
@@ -198,6 +198,13 @@ module Interfaces
     dive5(::Penguin, ::Int) = 5                 # Int >: Bottom
     dive6(::Penguin, ::Int) = 6                 # not Int >: Number
 
+    abstract type Animal end
```

Comment:
Can we add a comment about what this setup is intended for? Also would be nice to reference issue #30 directly in the comment.

Reply:
done